src/test/java/org/jenkinsci/plugins/durabletask/BourneShellScriptTest.java

```diff
@@ -107,4 +109,17 @@ public void smokeTest() throws Exception {
         c.cleanup(ws);
     }
 
+    @Issue("JENKINS-27152")
+    @Test public void cleanWorkspace() throws Exception {
+        Controller c = new BourneShellScript("touch stuff && echo ---`ls -1a`---").launch(new EnvVars(), ws, launcher, listener);
+        while (c.exitStatus(ws, launcher) == null) {
+            Thread.sleep(100);
+        }
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        c.writeLog(ws, baos);
+        assertEquals(0, c.exitStatus(ws, launcher).intValue());
+        assertThat(baos.toString(), containsString("---. .. stuff---"));
+        c.cleanup(ws);
```

Comment:
The call to `cleanup` you mean? Nothing more than that the standard call in the expected method sequence proceeds without error.